app/config/ApplicationConfig.scala

```diff
@@ -78,6 +78,7 @@ class ApplicationConfig @Inject()(configuration: Configuration, runMode: RunMode
   lazy val fileUploadEnabled: Boolean = loadBooleanConfig("featureFlags.fileUploadEnabled")
   lazy val downtimePageEnabled: Boolean = loadBooleanConfig("featureFlags.downtimePageEnabled")
   lazy val dvrEnabled: Boolean = loadBooleanConfig("featureFlags.dvrEnabled")
+  lazy val newAppointAgentJourneyEnabled: Boolean = loadBooleanConfig("featureFlags.newAppointAgentJourneyEnabled")
```

Comment:
new stuff should be assign, right?

Reply:
i used this name because the flag is already called "newAppointAgentJourneyEnabled" in the dashboard. I wanted to keep the same name